backend/src/handlers/alert.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from fastapi import APIRouter, Form, Request
+from fastapi.responses import JSONResponse
+from src.model.alert import Alert
+
+router = APIRouter(prefix="/alerts", tags=["alerts"])
+
+
+@router.post("/")
+def post_alert(
+    req: Request, text: str = Form(), start_time: int = Form(), end_time: int = Form()
+) -> JSONResponse:
+    with req.app.state.db.session() as session:
+        dt_start_time = datetime.datetime.fromtimestamp(start_time)
+        dt_end_time = datetime.datetime.fromtimestamp(end_time)
+
+        alert = Alert(text=text, start_datetime=dt_start_time, end_datetime=dt_end_time)
```

Comment:
nit: `new_alert` is clearer

Reply:
I somewhat disagree, the fact that we're instantiating an Alert sort of implies to me that it's a *new* object of the alert class. Happy to change it though if people want.

Reply:
My convention with the ridership route was to call it `new_ridership`, hence why I wanted to change it so that it's consistent.

Reply:
No, Alert/Ridership/etc are just classes. This definitely is opinion, but I think new_ridership is not what we would want. I think we would want Ridership. instantiation of the class is implying it's new.